ember/app/ui/profile/index/controller.js

```diff
@@ -3,24 +3,9 @@ import { action } from "@ember/object";
 import { inject as service } from "@ember/service";
 import { tracked } from "@glimmer/tracking";
 import { restartableTask } from "ember-concurrency";
-import QueryParams from "ember-parachute";
 
-const queryParams = new QueryParams({
-  // Convert the tab number to a string so its human readable
-  activeTab: {
-    defaultValue: "identity",
-    serialize(value) {
-      return value ? "memberships" : "identity";
-    },
-    deserialize(value) {
-      return value === "identity" ? 0 : 1;
-    },
-  },
-});
-
-export default class ProfileIndexController extends Controller.extend(
-  queryParams.Mixin
-) {
+export default class ProfileIndexController extends Controller {
+  queryParams = ["activeTab"];
```

Comment:
No more `defaultValue: "identity"` needed?

Reply:
No more need, as we use 0 or 1